sklearn/tests/test_metaestimators.py

```diff
@@ -44,6 +49,38 @@ def __init__(self, name, construct, skip_methods=(),
                                 'predict'])
 ]
 
+TESTED_META = [
+    # pipeline
+    # this fails because tuple is converted to list in fit:
+    Pipeline((('ss', StandardScaler()),)),
+    # all pipelines fail because they don't clone:
+    Pipeline([('ss', StandardScaler())]),
+    make_pipeline(StandardScaler(), LogisticRegression()),
+    # union
+    make_union(StandardScaler()),
+    # union and pipeline
+    make_pipeline(make_union(PCA(), StandardScaler()),
+                  LogisticRegression()),
+    # pipeline with clustering
+    make_pipeline(KMeans(random_state=0)),
+    # SelectFromModel
+    make_pipeline(SelectFromModel(LogisticRegression(), threshold=-np.inf),
+                  LogisticRegression()),
+    # grid-search
+    GridSearchCV(LogisticRegression(), {'C': [0.1, 1]}, cv=2),
+    # will fail tragically
+    # make_pipeline(StandardScaler(), None)
```

Comment:
Is this failing because of how `None` or `'passthrough'` does not support `_get_tags`?